---
title: 'Structure of co-expression networks of Bifidobacterium species in response to human milk oligosaccharides'

# Authors
# If you created a profile for a user (e.g. the default `admin` user), write the username (folder name) here
# and it will be replaced with their full name and linked to their profile.
authors:
  - Kevin J. Gonz√°lez-Morelo
  - admin
  - Ernesto Perez-Rueda
  - Daniel Garrido


date: '2023-01-26T00:00:00Z'
doi: 'https://doi.org/10.3389/fmolb.2023.1040721'

# Schedule page publish date (NOT publication's date).
publishDate: '2023-01-26T00:00:00Z'

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ['2']

# Publication name and optional abbreviated publication name.
publication: Frontiers in Molecular Biosciences
publication_short: Frontiers in Molecular Biosciences

abstract: 'Biological systems respond to environmental perturbations and a large diversity of compounds through gene interactions, and these genetic factors comprise complex networks. Experimental information from transcriptomic studies has allowed the identification of gene networks that contribute to our understanding of microbial adaptations. In this study, we analyzed the gene co-expression networks of three Bifidobacterium species in response to different types of human milk oligosaccharides (HMO) using weighted gene co-expression analysis (WGCNA). RNA-seq data obtained from Geo Datasets were obtained for Bifidobacterium longum subsp. Infantis, Bifidobacterium bifidum and Bifidobacterium longum subsp. Longum. Between 10 and 20 co-expressing modules were obtained for each dataset. HMO-associated genes appeared in the modules with more genes for B. infantis and B. bifidum, in contrast with B. longum. Hub genes were identified in each module, and in general they participated in conserved essential processes. Certain modules were differentially enriched with LacI-like transcription factors, and others with certain metabolic pathways such as the biosynthesis of secondary metabolites. The three Bifidobacterium transcriptomes showed distinct regulation patterns for HMO utilization. HMO-associated genes in B. infantis co-expressed in two modules according to their participation in galactose or N-Acetylglucosamine utilization. Instead, B. bifidum showed a less structured co-expression of genes participating in HMO utilization. Finally, this category of genes in B. longum clustered in a small module, indicating a lack of co-expression with main cell processes and suggesting a recent acquisition. This study highlights distinct co-expression architectures in these bifidobacterial genomes during HMO consumption, and contributes to understanding gene regulation and co-expression in these species of the gut microbiome.'

# Summary. An optional shortened abstract.
#summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags: [Bifidobacterium, Data mining, GCN]

# Display this page in the Featured widget?
featured: true

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

url_pdf: './gonzalez2023.pdf'
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
image:
  caption: 'Bacteria co-expression modules in (A) B. infantis ATCC 15697, (B) B. bifidum SC55, and (C) B. longum subsp. longum SC596.'
  focal_point: ''
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects:
  - GCN


---

